states.json

```diff
@@ -5,8 +5,7 @@
   "AR": "Arkansas",
   "CA": "California",
   "CO": "Colorado",
-  "CT": "Connecticut",
-  "DE": "Delaware",
+  "CT": "Connecticut", "DE": "Delaware",
```

Comment:
revert this, and LGTM

Reply:
Whoops, don't know why that happened. Pushed Fix! 👍